### ISSUE TITLE
Table is missing structural headers (PDF)

### ISSUE THREAD

**Issue by dannychapman** [department-of-veterans-affairs/vets-website#859]
https://www.vets.gov/playbook/downloads/VBA_Graphic_Styleguide.pdf
